plugins/resource_management/app/controllers/resource_management/automation_controller.rb

```diff
@@ -18,7 +18,7 @@ def dump_data
       legacy_names = {
         :compute        => 'compute',
         :dns            => 'dns',
-        :loadbalancing  => 'loadbalancing',
+        :loadbalancing  => 'lbaas2',
```

Comment:
@majewsky, @hgw77  would you check please if this change is correct.

Reply:
@majewsky please check this part if it makes any side effects. Thx

Reply:
This controller has not been in use since 2017, so no.